linters/codespell/plugin.yaml

```diff
@@ -12,6 +12,7 @@ lint:
       files: [ALL]
       direct_configs: [.codespellrc]
       suggest_if: config_present
+      description: Check for spelling mistakes in your code
```

Comment:
nit: Check for common misspellings in your code (mainly for **us** to differentiate codespell from cspell)